content/en/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md

```diff
@@ -552,16 +560,20 @@ Additionally, the following restrictions are applied to the schema:
   - `writeOnly`,
   - `xml`,
   - `$ref`.
-- The field `uniqueItems` cannot be set to _true_.
-- The field `additionalProperties` cannot be set to _false_.
+- The field `uniqueItems` cannot be set to `true`.
+- The field `additionalProperties` cannot be set to `false`.
 - The field `additionalProperties` is mutually exclusive with `properties`.
 
 These fields can only be set with specific features enabled:
-
-You can also use [Validation Schema Defaulting](/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#defaulting) to apply default values.
+ 
+- `default`: can be set for `apiextensions.k8s.io/v1` CustomResourceDefinitions.
+   Defaulting is in GA since 1.17 (beta since 1.16 with the `CustomResourceDefaulting`
+   [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+    enabled, which is the case automatically for many clusters for beta features).
+   Compare [Validation Schema Defaulting](#defaulting).
```

Comment:
Hi @tengqm . This is a suggestion/nit:

What do you think about moving the information about "Defaulting" to the Defaulting section in this page?
Since there is one field, could this sentence be removed:
"These fields can only be set with specific features enabled:"

Then, add the first sentence as the last list item:
The field `default` can be set for CustomResourceDefinitions that enable `apiextensions.k8s.io/v1`.
Setting the `default` field is similar to enabling [Defaulting in the validation schema](#defaulting). 

nit: Do you set this field to true?

nit: What do you think about editing the note:
See the [Structural schemas](...) section to read about other restrictions. OR
To read about other restrictions and CustomResourceDefinition features, see the section on [Structural schemas](...).

Reply:
Line 569-573 had been in the original text and was removed in a previous PR. I'm adding it back because _validation_ and _defaulting_ are two different tasks. This subsection is discussing what are the allowed schema fields during validation. The `default` field may be invalid when you using an older version of the CRD resource and /or you have the feature gate disabled. Defaulting, as another topic, is something related to this `default` key.

Revising the "These fields can only be set with specific features enabled:" statement is something I can do.

Will revise the note as per suggested.